command fails: mvn solr-runner:start -pl ubo-webapp

[ERROR] Failed to execute goal org.mycore.plugins:solr-runner-maven-plugin:1.1:start (default-cli) on project ubo-webapp: Error while downloading!: http://mirror.23media.de/apache/lucene/solr/8.6.2/solr-8.6.2.zip -> [Help 1]

--- a/ubo-webapp/pom.xml
+++ b/ubo-webapp/pom.xml
@@ -25,7 +25,7 @@
     <cargo.tomcat.ajp.port>8292</cargo.tomcat.ajp.port>
     <cargo.version>1.7.8</cargo.version>
     <jetty.version>9.4.26.v20200117</jetty.version>
-    <solr-runner.solrMirrorURL>http://mirror.23media.de/apache/lucene/solr/</solr-runner.solrMirrorURL>
+    <solr-runner.solrMirrorURL>https://archive.apache.org/dist/lucene/solr/</solr-runner.solrMirrorURL>
     <solr-runner.solrVersionString>8.6.2</solr-runner.solrVersionString>
     <solr.home>${user.config.dir}/${MCR.AppName}/data/solr</solr.home>
     <tomcat.version>9.0.34</tomcat.version>

